### PR TITLE
[merged] Fix atomic host man page

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by rhatdan** [comment]
@chuanchangjia PTAL

File: docs/atomic-host.1.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ can perform upgrades, rollbacks, and system state inspection.  It is used
 for implementations of the Project Atomic Host pattern.
 
 #NOTE
-The `host` subcommand is only available when `rpm-ostree` is installed.
+The `host` subcommand is only available on `atomic host` systems.
```
Comment (jlebon) [inline docs/atomic-host.1.md]: Maybe "on Atomic Host systems." instead? Otherwise it seems kinda recursive.

**Comment by rhatdan** [comment]
@jlebon Fixed

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**